config/settings.yml

```
@@ -541,6 +541,7 @@ flipper:
     - tony.williams@va.gov
     - travis.hilton@oddball.io
     - zurbergram@gmail.com
+    - ssuddath@governmentcio.com
```

```suggestion
    - ssuddath@governmentcio.com
    - tony.williams@va.gov
    - travis.hilton@oddball.io
    - zurbergram@gmail.com
```

Minor nitpick: a-z order